apps/aehyperchains/src/connectors/aehc_chain_sim_connector.erl

```diff
@@ -2,23 +2,80 @@
 
 -behaviour(aehc_connector).
 
--export([broadcast/3, verify/1, subscribe/1]).
+-behaviour(gen_server).
 
--spec broadcast(_From, _TxPayload, _To) ->
-                       binary().
-broadcast(From, TxPayload, To) ->
+%% API.
+
+-export([start_link/0]).
+
+%% gen_server.
+-export([init/1]).
+-export([handle_call/3]).
+-export([handle_cast/2]).
+-export([handle_info/2]).
+-export([terminate/2]).
+
+-export([send_tx/1, get_block/1]).
+
+%% API.
+
+id() -> 
+    ?MODULE.
+
+-spec start_link() ->
+                        {ok, pid()} | ingnore | {error, term()}.
+start_link() ->
+    Id = id(),
+    gen_server:start_link({local, Id}, ?MODULE, [], []).
+
+%%%===================================================================
+%%%  aehc_connector behaviour
+%%%===================================================================
+
+-spec send_tx(Tx::aetx:tx()) -> binary().
```

Comment:
Don't specify variable names in typespecs unless you have a very good reason to do so:
![image](https://user-images.githubusercontent.com/1352848/91422177-1b894080-e857-11ea-9923-82edb53107de.png)